### ISSUE TITLE
requirement problem

### ISSUE THREAD

**Issue by Vilsafur** [caffeinated/themes#54]
Hi,

I can't require with Laravel 5.4.9, I got this result :

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for caffeinated/themes ^3.0 -> satisfiable by caffeinated/themes[v3.0].
    - Conclusion: remove laravel/framework v5.4.9
    - Conclusion: don't install laravel/framework v5.4.9
    - caffeinated/themes v3.0 requires illuminate/view 5.3.* -> satisfiable by illuminate/view[v5.3.0, v5.3.16, v5.3.23, v5.3.4].
    - don't install illuminate/view v5.3.0|don't install laravel/framework v5.4.9
    - don't install illuminate/view v5.3.16|don't install laravel/framework v5.4.9
    - don't install illuminate/view v5.3.23|don't install laravel/framework v5.4.9
    - don't install illuminate/view v5.3.4|don't install laravel/framework v5.4.9
    - Installation request for laravel/framework (locked at v5.4.9, required as 5.4.*) -> satisfiable by laravel/framework[v5.4.9].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by UnderlineWords** [comment]
Pending at https://github.com/caffeinated/themes/issues/51

**Comment by Vilsafur** [comment]
Ok, thanks, 
How long to validate this pull-request?

**Comment by kaidesu** [comment]
v3.0.1 has been released with Laravel 5.4 support
